app/models/notification.rb

```diff
@@ -93,6 +93,7 @@ def send_new_badge_notification(notifiable)
     def send_reaction_notification(notifiable)
       return if notifiable.user_id == notifiable.reactable.user_id
       return if notifiable.points.negative?
+      return unless notifiable.reactable.receive_notifications
```

Comment:
Not sure if we wanted to mute comments *AND* reaction notifications. Personally think it's okay to mute _just_ comments, and if we want to go with that it'll be easy to remove the "mute reaction notifications" functionality.